packages/AdaptiveCards/Microsoft.Bot.Components.AdaptiveCards.csproj

```diff
@@ -0,0 +1,60 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Bot.Components.AdaptiveCard</PackageId>
```

Comment:
Rename to Microsoft.Bot.Components.AdaptiveCards (plural)

Reply:
good catch

---
In reply to: [609111342](https://github.com/microsoft/botframework-components/pull/862#discussion_r609111342) [](ancestors = 609111342)